Gemfile

```diff
@@ -13,21 +26,23 @@ group :test do
   gem 'addressable', '~> 2.3.8'
   gem 'backports'
   gem 'coveralls'
-  gem 'rack', '~> 1.2', :platforms => [:jruby_18, :jruby_19, :ruby_18, :ruby_19, :ruby_20, :ruby_21]
   gem 'rspec', '>= 3'
-  gem 'rubocop', '~> 0.50.0', :platforms => [:ruby_20, :ruby_21, :ruby_22, :ruby_23, :ruby_24] if RUBY_VERSION >= '2.0'
   gem 'simplecov', '>= 0.9'
 
-  platforms :jruby_18, :ruby_18 do
+  # For old ruby, restrict these gems to old version
+  if ruby_version < Gem::Version.new('1.9')
     gem 'mime-types', '~> 1.25'
     gem 'rest-client', '~> 1.6.0'
   end
-
-  platforms :ruby_18, :ruby_19 do
+  if ruby_engine == 'ruby' && ruby_version < Gem::Version.new('2.0')
     gem 'json', '< 2.0'
     gem 'term-ansicolor', '< 1.4.0'
     gem 'tins', '< 1.7'
   end
+  gem 'rack', '~> 1.2' if ruby_version < Gem::Version.new('2.2')
+
+  # Rubocop only works on new ruby
+  gem 'rubocop', '>= 0.37' if ruby_version >= Gem::Version.new('2.1')
```

Comment:
I’m not sure why RuboCop was reverted from the locked down version `~> 0.50.0` to this much larger range of versions? This is the opposite of what was fixed in #322.

Reply:
Great point.  I will see what happens when I set it back to `'~> 0.50.0'`

Reply:
I’m working on a PR :-)

Reply:
Bad rebase, I think. Sorry!